spec/features/admin/merchants/edit_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+RSpec.describe 'As an admin,' do
+  describe "When I visit a merchant's admin edit page" do
+    it "I see a form filled in with the existing merchant attribute information" do
+      visit admin_merchants_path
+
+      expect(page).to have_link("Schroeder-Jerde")
+      click_link "Schroeder-Jerde"
+      expect(page.current_path).to eq admin_merchant_path(1)
+
+      expect(page).to have_link("Update Merchant")
+      click_link "Update Merchant"
+      expect(page.current_path).to eq edit_admin_merchant_path(1)
+
+      within("#update_merchant") do
+        merchant_name = find("#merchant_name")
+
+        expect(merchant_name.value).to eq "Schroeder-Jerde"
+      end
+    end
+
+    it "When I update the information in the form and I click ‘submit’, Then I am redirected back to the merchant's admin show page where I see the updated information" do
+      visit admin_merchants_path
+
+      expect(page).to have_link("Schroeder-Jerde")
+      click_link "Schroeder-Jerde"
+      expect(page.current_path).to eq admin_merchant_path(1)
+
+      expect(page).to have_link("Update Merchant")
+      click_link "Update Merchant"
+      expect(page.current_path).to eq edit_admin_merchant_path(1)
+
+      within("#update_merchant") do
+        merchant_name = find("#merchant_name")
+
+        expect(merchant_name.value).to eq "Schroeder-Jerde"
+        fill_in "merchant_name", with: "test_merchant"
+        click_on "Update Merchant"
+      end
+
+      expect(page.current_path).to eq admin_merchant_path(1)
+      expect(page).to have_content("test_merchant")
+      expect(page).to_not have_content("Schroeder-Jerde")
+    end
+    end
+
+    it "I see a flash message stating that the information has been successfully updated." do
+      visit admin_merchants_path
```

Comment:
Probably need to add some sad path testing (I also need to add on the Merchants Items Side)

Reply:
Let me do this now!